### DISCUSSION TITLE
Sticky/Frozen column with React Spectrum TableView component

### DISCUSSION THREAD

**Discussion by cheema-corellian** [adobe/react-spectrum#6523] [Q&A]
We are currently using React Spectrum TableView component and quite happy with it.  I've been asked to make the first column sticky/frozen.  There are use cases where a user would want to add additional columns to the table even if they do not have enough horizontal screen space to fit everything.  This will add a horizontal scroll.  And this is when a sticky column would be nice.

I initially thought that setting the isRowHeader prop on Column component might do it.  But, it appears not to.

Is there another way to do this or is this not possible with this component?

**Comment by snowystinger** [comment]
We don't currently support this.
See https://github.com/adobe/react-spectrum/issues/6080#issuecomment-2012281586
